<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCalibration</class>
 <widget class="QDialog" name="DialogCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>相机标定</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,4">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="verticalSpacing">
         <number>25</number>
        </property>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_cam"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>相机</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget
{
background-color: rgb(42, 52, 64);
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="2,4,2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>定位结果</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,1,0">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_6" rowstretch="0,0">
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>机械坐标x</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>机械坐标y</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_robotX">
                 <property name="minimum">
                  <double>-99999.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>99999.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_robotY">
                 <property name="minimum">
                  <double>-99999.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>99999.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>相机坐标x</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_camX">
                 <property name="minimum">
                  <double>-99999.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>99999.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>相机坐标y</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_camY">
                 <property name="minimum">
                  <double>-99999.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>99999.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButton_record">
               <property name="text">
                <string>记录</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>标定数据记录</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="2">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>机械坐标x</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>机械坐标y</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>相机坐标x</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>相机坐标y</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTableWidget" name="tableWidget">
             <property name="styleSheet">
              <string notr="true">QTableWidget QHeaderView::section{
	color: #ffffff;
	background-color: #263851;
	border:1px solid #7B95AF;
}

QTableWidget {
	
	font: 14pt;
}</string>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>138</number>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>机械坐标x</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>机械坐标y</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>相机坐标x</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>相机坐标y</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_clear">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>标定结果</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10" rowstretch="1,0,0,0,0,0" columnstretch="1,0">
           <item row="4" column="1">
            <widget class="QPushButton" name="pushButton_calibration">
             <property name="text">
              <string>标定</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_9" columnstretch="0,0,0,0,0,0">
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>E00</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_H00">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>E10</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_H10">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>E01</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>E11</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_H01">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_H11">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>E02</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_H02">
               <property name="minimum">
                <double>-99999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>E12</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_H12">
               <property name="minimum">
                <double>-99999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="pushButton_calibration_check">
             <property name="text">
              <string>标定验证</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
